src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -327,6 +336,32 @@ export default defineComponent({
     this.checkIfWatched()
   },
   methods: {
+    getDeArrowTitle: async function() {
+      const data = await this.getDeArrowDataEntry()
+      return data.title
+    },
+
+    getDeArrowDataEntry: async function() {
+      // Read from local cache or remote
+      // Write to cache if read from remote
+      if (!this.useDeArrowTitles) { return null }
+
+      if (this.deArrowCache) { return this.deArrowCache }
+
+      const videoId = this.id
+      const data = await deArrowData(this.id)
+      const cacheData = { videoId, title: null }
+      if (Array.isArray(data?.titles)) {
+        if (data.titles.length > 0 && (data.titles[0].locked || data.titles[0].votes > 0)) {
+          cacheData.title = data.titles[0].title
+        }
+      }
```

Comment:
Any reason for nesting? (Too long?)

Also should we search for first locked title if absent search for most voted title?
```suggestion
      if (Array.isArray(data?.titles) && data.titles.length > 0 && (data.titles[0].locked || data.titles[0].votes > 0)) {
        cacheData.title = data.titles[0].title
      }
```